US-CBP/cbp-theme
# Added bootstrap carousel component and made some updates to index.html

What's this PR do?
Adds a new bootstrap carousel widget and fixes the "back-to-top" functionality of the up chevrons used throughout the page.

Where should the reviewer start?
Can begin by inspecting the new carousel's functionality and look on the index.html page and clicking on the up chevrons used throughout.

How should this be manually tested?
Can manually check the new carousel's functionality and look on the index.html page and clicking on the up chevrons used throughout.

Any background context you want to provide?
We are adding this new carousel offering in responses to a client's recent request.

What are the relevant tickets?
n/a

Screenshots (if appropriate)
We added the 'margin: auto' rule to allow for any width to be used and avoid looking like below:

![image](https://user-images.githubusercontent.com/24257663/58832540-27b72b80-861d-11e9-9552-fbd02e194532.png)

So, now any width less than the carousel's container will auto center like below;

![image](https://user-images.githubusercontent.com/24257663/58832857-dfe4d400-861d-11e9-863f-ae1a7d7d45b5.png)


Also, we are providing a second carousel option displaying images at full width of the container, which applies the 'width: 100%' rule if used.  This will give another choice for our users.

![image](https://user-images.githubusercontent.com/24257663/58832949-06a30a80-861e-11e9-9bd4-56b163c4289f.png)


Questions:
Is there a blog post?
n/a
Does the knowledge base need an update?
do not think so at this point.
